doc/src/records/gain.md

```diff
@@ -125,6 +138,17 @@ Name of a DHWSYS whose water use modulates gnPower.  For example, electricity us
 ----------- ------------------- ---------------------- -------------- -----------------
             *name of a DHWSYS*  no DHWSYS/GAIN linkage            No         constant
 
+**gnCtrlDHWMETER=*integer***
+
+Controls the DHWMETER. Allows gains to track water usage such as dishwashers, clothes washers, etc.
```

Comment:
This is actually the DHWMETER that controls the gain (not the other way around as it currently reads).